scripts/cangen/cangen/can_generator.py

```diff
@@ -226,6 +224,7 @@ def generate_code(bus: Bus, config: Config):
     dbc_file = bus.dbc_file_path
     our_node = config.node
     bus_name = bus.bus_name
+    output_dir = Config.output_dir
```

Comment:
few things here:

1. `Config` is a class. You should be accessing the `config` variable.
2. The variable is not called `.output_dir` in the class definition
3. You don't need to make a variable here. Just directly use the class member in the `os.path.join` statements below